### DISCUSSION TITLE
Controller is detected by Windows but not DsHidMini

### DISCUSSION THREAD

**Discussion by namenamenam** [nefarius/DsHidMini#168] [Support]
I can connect my DS3 to my computer through a USB port (worth mentioning that I'm using a hub) and Windows plays the normal connected device sound (default Windows sound), my controller indicates it's charging (using the stock animation), but DsHidMini doesn't show the controller, elevated or not. Oddly enough, pressing the PS button plays the connection sound too (turning on the DS3 has all lights blinking quickly, but they never stop), and holding the PS button to shut off the controller plays the disconnect sound (and stops the blinking). It's worth mentioning that the copies of DsHidMini and BthPS3 are both the latest prebuilt versions. I am not using DS4Windows. What exactly is causing my problem, and how do I fix it?

**Comment by nefarius** [comment]
If you want meaningful help, whip out all the missing information; OS and driver versions used, have you followed the troubleshooting guides and FAQs on the site? How's Device Manager looking? Screenshots? Info, info, info!

**Reply by namenamenam** [reply]
I'm running Windows 10 version 21H1, DsHidMini 2.2.282, BthPS3 2.1.153. Upon connecting the controller, **a new device labelled HID-compliant game controller appears**. So maybe it is working, but it still won't appear in the DsHidMini window; it continues saying "No compatible devices found". From what I can gather on the troubleshooting pages, I'm pretty sure I'm supposed to "just do it". It's worth noting that BthPS3 was installed **before** DsHidMini.

Screenshots:
https://drive.google.com/file/d/18A9YaesB-BNDBlDigLevPRvezrDeq_mG/view?usp=sharing
https://drive.google.com/file/d/1qgxBM5Ysji-yxYgGVfwWgWC1vaz-MVtv/view?usp=sharing

**Reply by nefarius** [reply]
It's supposed to appear in a new group named Nefarius HID Devices, if not, it's not running under DsHidMini. Check out the troubleshooting section, that tells you how to verify.
